test/models/chan.js

```diff
@@ -32,9 +32,7 @@ describe("Chan", function() {
 		};
 
 		var getUserNames = function(chan) {
-			return chan.users.map(function(u) {
-				return u.name;
-			});
+			return chan.users.map(u => u.nick);
```

Comment:
Can we stick () around "u" because it's one of those things like doing {} round a 1 line if. It's not "wrong" but it can stop us having problemsi n the future if we add stuff to it.